crypto/rand/rand_egd.c

```diff
@@ -100,6 +167,18 @@ int RAND_query_egd_bytes(const char *path, unsigned char *buf, int bytes)
             /* No error, try again */
             break;
         default:
+#if defined(OPENSSL_SYSNAME_TANDEM)
+              if (hpns_connect_attempt == 0)
```

Comment:
does this ever need to reset to zero?

Reply:
> 
> 
> does this ever need to reset to zero?

I don't think it is reset. It is a first attempt. I think it is used as a boolean but I have to get this answered by someone else on the team.